Assets/Scripts/Player/Commands/JumpCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
```

Comment:
Isn't all of this Player stuff part of Hugo's PR that was merged? Why is it here??

Reply:
Because I had to change some stuff (fixes)

Reply:
it shows as though everything is completely new, though. Odd.